Delicate-Butterflies/Bangazon-Cli
# Mb prompt complete order#6

Related Issue: #6 #33 

***to check out this branch after fetching, please put quotes around branch name***

Description:
1. user can complete a customer order by adding a payment type
2. displayWelcome() is updated to our specifications rather than boilerplate

Notes for Readme Info:
js doc annotations added

Chai/Mocha Tests added:
complete-order.test.js

Any other PR Testing Steps:
for cli -
1. choose an active customer and follow steps from option 6 until you've added a payment type to that customer's order
2. choose an active customer and follow steps from option 6 and choose no when prompted if you want to complete order
3. create a new customer who has no orders and attempt to choose option 6
4. make sure welcome display menu is correct
for tests - 
5. confirm they all pass
6. let me know if there are additional tests you would like to see
for documentation - 
7. confirm correct format
